.github/workflows/tyk-headless.yaml

```diff
@@ -0,0 +1,52 @@
+name: Tyk Headless Push Helm
+on: push
+jobs:
+  push:
+    name: Tyk Headless Push CS
+    runs-on: ubuntu-latest
+    branches:
+      - main
+      - release-**
+    tag: 'v*-headless'
+    paths:
+      - 'tyk-headless/**'
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set env
+        run: |
+          VER=$(sed 's/[^0-9.]//g' <<< GITHUB_REF##*/)
+          echo "RELEASE_VERSION=${VER}" >> $GITHUB_ENV
+
+      - name: Install Helm, create dirs
```

Comment:
Isn't it easier to use [this](https://github.com/marketplace/actions/helm-tool-installer) action instead?

Reply:
Could be a ggood addition, gotta try it

Reply:
@gwotg Would be nice to use it like in the other workflows, e.g. here:
https://github.com/TykTechnologies/tyk-helm-chart/blob/5923a801bc407fedea34f0b5ce27d4d9a65b933c/.github/workflows/smoke-test.yml#L100-L103